apps/cnp/plum-frontend/demo.yaml

```diff
@@ -10,4 +10,4 @@ spec:
       replicas: 4
       ingressHost: plum.demo.platform.hmcts.net
       spotInstances:
-        enabled: true
\ No newline at end of file
+        enabled: false
```

Comment:
please consider Martin's comment about removing `spotInstances` in environment specific values files @ryanmcalary11